engine/src/main/java/org/terasology/logic/console/commands/CoreCommands.java

```diff
@@ -401,6 +404,75 @@ public String spawnBlock(@Sender EntityRef sender, @CommandParam("blockName") St
         return "Spawned block.";
     }
 
+
+
```

Comment:
Some extra newlines here. The indentation for the entire function is slightly off at places - if you're running Eclipse/IDEA, set your code formatter to Terasology's (`docs/formatter.xml`) and autoformat.